plugins/arDominionB5Plugin/modules/accession/templates/indexSuccess.php

```diff
@@ -9,9 +9,11 @@
 
 <?php slot('context-menu'); ?>
 
-  <?php if (check_field_visibility('app_element_visibility_physical_storage')) { ?>
-    <?php echo get_component('physicalobject', 'contextMenu', ['resource' => $resource]); ?>
-  <?php } ?>
+  <nav role="navigation">
```

Comment:
I'm not sure how this will be interpreted by screen readers if it's empty, I'd move the nav inside the if, just in case.

Reply:
You're right, fixed!